### PR TITLE
compton: build and install man pages, add missing license

### PR DESCRIPTION
**Pull request by yshui**

### CODE REVIEW

File: srcpkgs/compton/template [hunk 1]
```diff
@@ -21,5 +22,6 @@ pre_build() {
 
 post_install() {
 	vlicense LICENSES/MIT
+	vlicense LICENSES/MPL-2.0
```
Comment (maxice8) [inline srcpkgs/compton/template]: it isn't required to provide the MPL license file

Comment (yshui) [reply srcpkgs/compton/template]: Fixed.

**Comment by yshui** [comment]
@maxice8 I looked into this a bit. MPL2 section 3.4 says:

```
You may not remove or alter the substance of any license notices (including copyright
notices, patent notices, disclaimers of warranty, or limitations of liability) contained 
within the Source Code Form of the Covered Software.
```

Doesn't that mean we have to ship the MPL-2.0 file?

**Comment by maxice8** [comment]
On Tue, Nov 27, 2018 at 10:53:41PM +0000, yshui wrote:
> @maxice8 I looked into this a bit. MPL2 section 3.4 says:
> 
> ```
> You may not remove or alter the substance of any license notices (including copyright
> notices, patent notices, disclaimers of warranty, or limitations of liability) contained 
> within the Source Code Form of the Covered Software.
> ```
> 
> Doesn't that mean we have to ship the MPL-2.0 file?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/5126#issuecomment-442249432

No

**Pull request merged**